Add NotEmptyFilter (#1207)
sonata-project/SonataDoctrineORMAdminBundle: docs/reference/filter_field_definition.rst

        }
    }

> This filter has an additional option ``inverse`` in order to be used as a ``NotEmptyFilter``.

Reviewer: ```suggestion
The ``inverse`` option can be used to filter values that are not empty.
```
BTW, we should add an example using this option.

Author: I made the change.

But I don't think an example would be useful

This option can be used like any other filter option.
```
final class PostAdmin extends AbstractAdmin
    {
        protected function configureDatagridFilters(DatagridMapper $datagridMapper)
        {
            $datagridMapper
                ->add('deleted', EmptyFilter::class, ['field_name' => 'deletedAt', 'inverse' => true]);
        }
    }
```
